skip downloading & building of Jellyfish, since it's provided via EasyBuild already
author: Kenneth Hoste (HPC-UGent)
--- src/CMakeLists.txt.orig	2015-10-09 17:43:38.341230754 +0200
+++ src/CMakeLists.txt	2015-10-09 17:43:59.221682024 +0200
@@ -109,7 +109,6 @@
     ${Boost_LIBRARIES} 
     ${GAT_SOURCE_DIR}/external/install/lib/libstaden-read.a
     ${ZLIB_LIBRARY} 
-    ${GAT_SOURCE_DIR}/external/install/lib/libjellyfish-2.0.a 
     ${GAT_SOURCE_DIR}/external/install/lib/libbwa.a
     m
     ${LIBLZMA_LIBRARIES}
--- CMakeLists.txt.orig	2015-10-12 20:28:35.975655281 +0200
+++ CMakeLists.txt	2015-10-12 20:28:44.735838142 +0200
@@ -353,21 +353,6 @@
     BUILD_IN_SOURCE TRUE
 )
 
-message("Build system will fetch and build Jellyfish")
-message("==================================================================")
-ExternalProject_Add(libjellyfish
-    DOWNLOAD_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external
-    URL ftp://ftp.genome.umd.edu/pub/jellyfish/jellyfish-2.1.3.tar.gz
-    SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external/jellyfish-2.1.3
-    INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/external/install
-    CONFIGURE_COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/external/jellyfish-2.1.3/configure --prefix=<INSTALL_DIR> CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} CXXFLAGS=${JELLYFISH_CXX_FLAGS}
-    BUILD_COMMAND ${MAKE} CC=${CMAKE_C_COMPILER} CXX=${CMAKE_CXX_COMPILER} CXXFLAGS=${JELLYFISH_CXX_FLAGS}
-    BUILD_IN_SOURCE 1
-    INSTALL_COMMAND make install && 
-                    cp config.h <INSTALL_DIR>/include/jellyfish-2.1.3/jellyfish/ &&
-                    cp config.h <INSTALL_DIR>/include/
-)
-
 find_package(TBB)
 
 ##
